### PR TITLE
Added per-sample denoised coverage output to gCNV

### PR DESCRIPTION
**Pull request by asmirnov239**
This PR adds two new files into the gCNV output directory - mean and the standard deviation of the denoised coverage, that is computed by drawing posterior samples from the computed model.

### CODE REVIEW

File: src/main/python/org/broadinstitute/hellbender/gcnvkernel/models/model_denoising_calling.py [hunk 1]
```diff
@@ -47,7 +47,8 @@ def __init__(self,
                  active_class_padding_hybrid_mode: int = 50000,
                  enable_bias_factors: bool = True,
                  enable_explicit_gc_bias_modeling: bool = False,
-                 disable_bias_factors_in_active_class: bool = False):
+                 disable_bias_factors_in_active_class: bool = False,
+                 denoised_counts_sampling_rounds: int = 250):
```
Comment (samuelklee) [inline src/main/python/org/broadinstitute/hellbender/gcnvkernel/models/model_denoising_calling.py]: If this parameter is going to be part of the config, should it be exposed (up through the python script, Java CLI, and WDL)?  It also feels a bit different from the other DenoisingModelConfig model parameters, in that it parameterizes the inference.  However, it doesn't really qualify as a HybridInferenceParameter, either.

Does output of the denoised coverage significantly affect runtime or memory?  If so, we should probably expose, and we can keep it here in the config.  If not, I'd be fine with setting an unexposed default, but I'd put it somewhere else besides the config.

File: src/main/python/org/broadinstitute/hellbender/gcnvkernel/io/io_commons.py [hunk 2]
```diff
@@ -12,6 +12,7 @@
 from . import io_consts
 from .._version import __version__ as gcnvkernel_version
 from ..models.fancy_model import GeneralizedContinuousModel
+from ..models.model_denoising_calling import DenoisingModel
```
[the remark this answers is not in the document]
Comment (asmirnov239) [reply src/main/python/org/broadinstitute/hellbender/gcnvkernel/io/io_commons.py]: No. Strangely, PyCharm does not let me know that..

File: src/main/python/org/broadinstitute/hellbender/gcnvkernel/io/io_consts.py [hunk 3]
```diff
@@ -32,6 +32,10 @@
 # column name for baseline copy-number file
 baseline_copy_number_column_name = "BASELINE_COPY_NUMBER"
 
+# column name for denoised copy-number files
```
[the remark this answers is not in the document]
Comment (samuelklee) [reply src/main/python/org/broadinstitute/hellbender/gcnvkernel/io/io_consts.py]: Don't forget to change the comment above.

File: src/main/python/org/broadinstitute/hellbender/gcnvkernel/io/io_denoising_calling.py [hunk 4]
```diff
@@ -151,6 +153,14 @@ def __call__(self):
         approx_var_set, approx_mu_map, approx_std_map = io_commons.extract_meanfield_posterior_parameters(
             self.denoising_model_approx)
 
+        # compute approximate denoised copy number ratios
+        denoising_copy_ratios_approx_generator = commons.get_sampling_generator_for_model_approximation(
```
Comment (samuelklee) [inline src/main/python/org/broadinstitute/hellbender/gcnvkernel/io/io_denoising_calling.py]: denoising -> denoised?  if not, disregard.

File: src/main/python/org/broadinstitute/hellbender/gcnvkernel/io/io_denoising_calling.py [hunk 5]
```diff
@@ -151,6 +153,14 @@ def __call__(self):
         approx_var_set, approx_mu_map, approx_std_map = io_commons.extract_meanfield_posterior_parameters(
             self.denoising_model_approx)
 
+        # compute approximate denoised copy number ratios
```
Comment (samuelklee) [inline src/main/python/org/broadinstitute/hellbender/gcnvkernel/io/io_denoising_calling.py]: copy number ratios -> copy ratios

**Comment by samuelklee** [comment]
Oops, one more thing.  Let's add these outputs to the relevant WDLs.

**Pull request merged**